### PR TITLE
[CBRD-20696] Correct disk space from createdb message

### PR DESCRIPTION
**Pull request by andrei14vl**
The database volume size from createdb mesage was set to its real value, according to the minimum number of sectors and the number of  pages  per sector.

### CODE REVIEW

File: src/storage/storage_common.h [hunk 1]
```diff
@@ -157,6 +157,7 @@ struct log_lsa
 /*
  * Sector
  */
+#define BOOT_VOLUME_MINSECTS 64
```
Comment (ribram) [inline src/storage/storage_common.h]: remove BOOT_VOLUME_MINSECTS and BOOT_VOLUME_MINPAGES.
remove unused function boot_max_pages_for_new_temp_volume.

create extern function in disk manager to return DISK_MIN_VOLUME_SECTS.

**Comment by andrei14vl** [comment]
I created the function `disk_sectors_to_hold_npages()` to simplify the conversion. 

Using `DISK_MIN_VOLUME_SECTS `was incorrect, since the ROUND_UP function is used in `disk_add_volume_extension `and `disk_format_first_volume`.

File: src/transaction/boot_sr.h [hunk 2]
```diff
@@ -103,7 +103,6 @@ extern VOLID boot_find_next_permanent_volid (THREAD_ENTRY * thread_p);
 extern int boot_reset_db_parm (THREAD_ENTRY * thread_p);
 extern DKNPAGES boot_max_pages_new_volume (void);
 extern DKNPAGES boot_max_pages_for_new_auto_volume_extension (void);
-extern DKNPAGES boot_max_pages_for_new_temp_volume (void);
 extern DKNPAGES boot_get_temp_temp_vol_max_npages (void);	/* todo: remove me */
```
Comment (eseokoh) [inline src/transaction/boot_sr.h]: The update looks very good to me. 

One observation I found is `boot_get_temp_temp_vol_max_npages` is also unused.
Would you also remove it?

**Comment by eseokoh** [comment]
OK. Thanks for the update :)

**Pull request merged**